markdownPreview/mermaid.ts

```diff
@@ -23,11 +23,7 @@ export function renderMermaidBlocksInElement(root: HTMLElement) {
     function renderMermaidElement(mermaidContainer: Element) {
         const id = `mermaid-${crypto.randomUUID()}`;
         const source = mermaidContainer.textContent ?? '';
-
-        const out = document.createElement('div');
-        out.id = id;
```

Comment:
Is it safe to remove this div? The contents of `mermaidContainer` gets overwritten on L35, but I'm not sure if it's used elsewhere too.

Reply:
Yes looks unused to me